pages/common/jello.md

```diff
@@ -0,0 +1,32 @@
+# jello
+
+> A command-line JSON processor using Python syntax.
+> More information: <https://github.com/kellyjonbrazil/jello>.
+
+- Pretty-print JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello`
+
+- Output a schema of JSON or JSON Lines data from stdin to stdout (useful for grep):
+
+`cat {{file.json}} | jello -s`
+
+- Output all elements from arrays (or all the values from objects) in JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello -l`
+
+- Output the first element in JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello _[0]`
+
+- Output the value of a given key of each element in JSON or JSON-Lines data from stdin to stdout:
+
+`cat {{file.json}} | jello '[i.{{key_name}} for i in _]'`
+
+- Output the value of multiple keys as a new JSON object (assuming the input JSON has the keys `key_name` and `other_key_name`):
+
+`cat {{file.json}} | jello '{"{{my_new_key}}": _.{{key_name}}, "{{my_other_key}}": _.{{other_key_name}}}'`
```

Comment:
@bl-ue do you think the extra curly brace will cause any render errors?

Reply:
For the node client, it renders beautifully!
<img width="1135" alt="Screen Shot 2021-06-11 at 4 23 21 PM" src="https://user-images.githubusercontent.com/54780737/121744757-594b1300-cad1-11eb-8be4-10e90ed2bf74.png">